content/docs-v1/install-uninstall.md

```diff
@@ -7,181 +7,72 @@ aliases:
+2. Download and install the **v18.12.1 release** of Node.js: [https://nodejs.org/download/release/v18.12.1/node-v18.12.1.pkg](https://nodejs.org/download/release/v18.12.1/node-v18.12.1.pkg). Node.js enables you to run javascipt on your computer.
+3. Back in your Terminal, install Quire with the command `npm install --global @thegetty/quire-cli@pre-release`
 
-    When complete, type `cd` into the Terminal to return to your {{< q-def "home directory">}}.
+    The Quire installation process may take a few minutes, during which time there will be messaging output in your Terminal. The only messages of any concern are those labeled as ERROR or ERR.
 
-    ```text
-    cd
-    ```
+4. Confirm installation by running the command `quire --help` in your Terminal. This should bring up a list of Quire commands. If you receive the message "command not found", this means installation was unsuccessful.
 
-5. **Quire** is operated through a {{< q-def "command-line interface" >}} (CLI) that enables you to create, preview and output publications using Terminal with commands like `quire new`, `quire preview` and `quire site`. You can learn more in the [*Quire Commands*](/docs-v1/quire-commands/) chapter of our guide. Copy and paste the following line into Terminal to install Quire:
+For installation troubleshooting, search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions).
 
-    ```text
-    npm install --global @thegetty/quire-cli
-    ```
-
-    The Quire installation process may take a minute or two, during which time there will be messaging output in Terminal. The only messages of any concern are those labeled as ERROR or ERR. These likely indicate a failed installation.
-
-    To verify installation, enter the command below. This will give you a list of [commands](/docs-v1/quire-commands/) that will help you get started using and navigating Quire. And if you get "command not found" it means it was not installed correctly.
-
-    ```text
-    quire --help
-    ```
-
-    Search or post to our [Discussions Forum](https://github.com/thegetty/quire/discussions) to troubleshoot installation issues.
-
-{{< q-class "box warning" >}}
-- If you are working on the macOS Monterey, you will need to also install **Python 2**. Follow this link: [https://www.python.org/downloads/release/python-2718/](https://www.python.org/downloads/release/python-2718/), click the "macOS 64-bit installer," and follow the install prompts.
+{{< q-class "box tip" >}}
+- If you are using the legacy version of Quire (v0), you will need to install the **v14.18.1 release** of Node.js: [https://nodejs.org/download/release/v14.18.1/node-v14.18.1.pkg](https://nodejs.org/download/release/v14.18.1/node-v14.18.1.pkg). Node Version Manager (NVM) is a software that can be used to toggle back-and-forth with Node.js versions. More inforamtion can be found at [https://github.com/nvm-sh/nvm](https://github.com/nvm-sh/nvm).
```

Comment:
I think something like this alternate wording might get more at what you're trying to say here?

> "If you will still be using the legacy version of Quire (v0) on past projects, you will need to continue to run those with **v14.18.1** of Node.js...."